velox/dwio/dwrf/common/Config.h

```diff
@@ -78,6 +78,48 @@ class Config : public config::ConfigBase {
   static Entry<uint64_t> RAW_DATA_SIZE_PER_BATCH;
   static Entry<bool> MAP_STATISTICS;
 
+  /// Maximum stripe size in orc writer.
+  static constexpr const char* kOrcWriterMaxStripeSize =
```

Comment:
nit: in the future, we should probably make these consistent with the upper case ones above. But nice that we are taking small(er) steps.